avocado/utils/process.py

```diff
@@ -147,13 +147,13 @@ def get_parent_pid(pid):
     """
     Returns the parent PID for the given process
 
-    TODO: this is currently Linux specific, and needs to implement
-    similar features for other platforms.
+    :note: This is currently Linux specific.
 
     :param pid: The PID of child process
     :returns: The parent PID
     :rtype: int
     """
+    # TODO: It needs to implement similar features for other platforms.
```

Comment:
pylint will complain soon about this (when we enable).

TODOs could be converted into issues here, on github. What do you think?

Reply:
@beraldoleal sound as a good idea. I will convert into issues then send a v2. Thanks!